examples/pending-order-reminder/Send Reminder on Approval Pending Orders.md

```diff
@@ -0,0 +1,35 @@
+# Send reminder on approval pending sales orders
+
+This example illustrates the use of the `sap:Client` in Ballerina to interact with S/4HANA APIs. Specifically, it demonstrates how to send a reminder email for sales orders that are pending approval.
+
+## Prerequisites
+
+### 1. Setup S/4HANA API
+
+Refer to the [Setup Guide](https://central.ballerina.io/ballerinax/sap/latest#setup-guide) for necessary credentials (hostname, username, password).
+
+### 2. Setup Gmail API
```

Comment:
```suggestion
### 2. Setup the Gmail API
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-sap/pull/10